oneflux_steps/Makefile

```diff
@@ -31,7 +31,7 @@ CC := gcc -O3
 MKDIR = mkdir -p
 
 # copy file command (verbose, keep file metadata)
-COPY = cp -av
+COPY = cp -v
```

Comment:
The comment on line 33 no longer matches the actual command!

Reply:
Will fix, thanks for pointing this out.